posts/2020-12-15-openliberty-realworld-example-app.adoc

```diff
@@ -14,15 +14,15 @@ David Shi <https://github.com/dshimo>
 
 
 
-In the sphere of web applications, a developer can easily feel overwhelmed by the large selection of options for a frontend, backend, and their supporting technologies.  You might strategically choose what's most popular or trending. Maybe you stumbled across stats showing that a certain framework performs a certain function particularly well. Or maybe, you just like the style. Choosing a framework can feel like buying new clothes; wouldn't you like to try out your choice before rocking it?
+In the sphere of web applications, a developer can easily feel overwhelmed by the large selection of options for a frontend, backend, and their supporting technologies.  You might strategically choose what's most popular or trending. Maybe you stumbled across stats showing that a certain framework performs a certain function particularly well. Or maybe, you just like its style. Choosing a framework can feel like buying new clothes; wouldn't you like to try out your choice before rocking it?
 
-link:https://github.com/gothinkster/realworld[RealWorld] serves as your fitting room for building "real world" applications. Created as an open source project, RealWorld enables users to choose any combination of frontend and backend to see how they work together in a sample app called link:https://demo.realworld.io/["Conduit"]. 
+Let link:https://github.com/gothinkster/realworld[RealWorld] serve as your fitting room for playing around and building "real world" applications. Created as an open source project, RealWorld enables users to choose any combination of frontend and backend to see how they work together in a sample app called link:https://demo.realworld.io/["Conduit"]. This provides developers a great sandbox environment for exploring new or familiar technologies. What if you're a frontend developer with no interest in backend, or vice versa? Since the frontend or backend can run independently of each other, you can for example, dedicate your efforts to finding the frontend that suits you without knowing any details of how the backend performs its tasks. RealWorld manages this by enforcing link:https://github.com/gothinkster/realworld/tree/master/spec#project-overview[API specs] for Conduit's endpoints.
```

Comment:
edit suggestions- avoid using "This" as the subject of a sentence. Can fix by answering "This what?". For example

This provides developers a great sandbox environment for exploring new or familiar technologies.
--->
This app provides developers a great sandbox environment for exploring new or familiar technologies.

RealWorld manages this by enforcing...
-->
RealWorld manages this  separation by enforcing